.github/workflows/fetch_all_tools.yaml

```diff
@@ -76,4 +77,6 @@ jobs:
           git add results
           git status
           git diff --quiet && git diff --staged --quiet || (git commit -m "fetch all tools bot  - step merge") 
-          git push
\ No newline at end of file
+          git push
```

Comment:
not sure if that makes sense, since the `GITHUB_API_KEY` is only used by our scripts.
to use a different PAT for the checkout action, we probably need to set 
```

```

Reply:
Ops, you're right. It should be fixed now.